pages/index.js

```diff
@@ -1,53 +1,68 @@
 import styled from "styled-components";
-import logoHomepage from "../assets/logoHomepage.png";
+import myLogo from "../assets/myLogo.png";
 import Image from "next/image";
 
 export default function HomePage() {
   return (
     <Wrapper>
       <StyledImage>
-        <Image
-          src={logoHomepage}
-          alt="Essensretter Willkommen Logo"
-          width={300}
-          height={300}
-        />
+        <Image src={myLogo} alt="Essensretter Logo" width={450} height={450} />
       </StyledImage>
       <StyledSection>
-        <StyledButton as="a" href={"/myDonations"}>
+        <StyledDonationButton as="a" href={"/myDonations"}>
```

Comment:
Usually this would be a nice solution to reuse a component, but I do not understand why you do not style a link instead. It also recommended to work the next/link component as this prevents the rerendering of the whole app.